app/src/main/java/com/android/streetworkapp/ui/park/ParkOverview.kt

```diff
@@ -130,14 +144,21 @@ fun ImageTitle(image: Painter?, title: String) {
  * @param park The park data to display.
  */
 @Composable
-fun ParkDetails(park: Park) {
+fun ParkDetails(park: Park, showRatingDialog: MutableState<Boolean>) {
   Column(modifier = Modifier.testTag("parkDetails")) {
     Text(
         text = "Details",
         fontSize = 24.sp,
         fontWeight = FontWeight.Bold,
         modifier = Modifier.padding(start = 16.dp, top = 6.dp, bottom = 2.dp))
-    RatingComponent(rating = park.rating.toInt(), park.nbrRating) // Round the rating
+
+    Row(modifier = Modifier.fillMaxWidth()) {
+      RatingComponent(rating = park.rating.toInt(), park.nbrRating) // Round the rating
+
+      // TODO: Check if the user has already rated the park and hide the button if true
```

Comment:
Side note - Currently I don't think it's possible to know if a user has already rated a park. I know this is not the purpose of this PR but I wanted to mention it. No need to reply to this comment.